buildSrc/src/main/java/kotlin-multiplatform.gradle.kts

```diff
@@ -9,4 +9,25 @@ extensions.getByType(JavaPluginExtension::class).apply {
   targetCompatibility = JavaVersion.VERSION_1_8
 }
 
+kotlin {
```

Comment:
glad you added this here. 👍

Reply:
I learnt about precompiled scripts from this project so thank you folks!